pennylane/beta/queuing/measure.py

```diff
@@ -61,6 +78,97 @@ def __init__(self, return_type, obs=None, wires=None):
         self.diagonalizing_gates = lambda: []
         self.data = []
 
+        # Queue the measurement process
+        self.queue()
+
+    @property
+    def wires(self):
+        r"""The wires the measurement process acts on."""
+        if self.obs is not None:
+            return self.obs.wires
+        return self._wires
+
+    @property
+    def eigvals(self):
+        r"""Eigenvalues associated with the measurement process.
+
+        If the measurement process has an associated observable,
+        the eigenvalues will correspond to this observable. Otherwise,
+        they will be the eigenvalues provided when the measurement
+        process was instantiated.
+
+        Note that the eigenvalues are not guaranteed to be in any
+        particular order.
+
+        **Example:**
+
+        >>> m = MeasurementProcess(Expectation, obs=qml.PauliX(wires=1))
+        >>> U.eigvals
+        >>> array([1, -1])
```

Comment:
Looks like there are typos in this docstring @josh146

Reply:
🤕

Reply:
Will fix it in the latest PR. Thanks!